ontology/chain-sequence.tsv

```diff
@@ -248,7 +248,7 @@ HLA-DQA1*01:04 chain	DQA1*01:04:01:01	IMGT/HLA	HLA00605	MILNKALLLGALALTTMMSPCGGE
 HLA-DQA1*02:01 chain	DQA1*02:01	IMGT/HLA	HLA00607	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQFTHEFDGDEEFYVDLERKETVWKLPLFHRLRFDPQFALTNIAVLKHNLNILIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPAPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
 HLA-DQA1*03:01 chain	DQA1*03:01:01	IMGT/HLA	HLA00608	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSADEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
 HLA-DQA1*03:02 chain	DQA1*03:02	IMGT/HLA	HLA00610	MILNKALMLGALALTTVTSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
-HLA-DQA1*03:03 chain	DQA1*03:03:01	IMGT/HLA	HLA00611	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
+HLA-DQA1*03:03 chain	DQA1*03:03:01:01	IMGT/HLA	HLA00611	MILNKALMLGALALTTVMSPCGGEDIVADHVASYGVNLYQSYGPSGQYSHEFDGDEEFYVDLERKETVWQLPLFRRFRRFDPQFALTNIAVLKHNLNIVIKRSNSTAATNEVPEVTVFSKSPVTLGQPNTLICLVDNIFPPVVNITWLSNGHSVTEGVSETSFLSKSDHSFFKISYLTFLPSDDEIYDCKVEHWGLDEPLLKHWEPEIPTPMSELTETVVCALGLSVGLVGIVVGTVLIIRGLRSVGASRHQGPL
```

Comment:
Hmm. Why has this changed?

Reply:
I'm not sure. I ran:
1. `make update-tsv`
2. `make update-iedb`
3. `make update-seqs`

Reply:
Ok